packages/builders/src/messages/embed/UnsafeEmbed.ts

```diff
@@ -315,6 +325,13 @@ export class UnsafeEmbed {
 		return { ...this.data };
 	}
 
+	public equals(other: UnsafeEmbed | APIEmbed) {
+		if (other instanceof UnsafeEmbed) {
+			return isEqual(other.data, this.data);
```

Comment:
This will check for hight, weight and all in thumbnail, image which was removed in v13.3 iirc

Reply:
Embeds aren't people! It's checking for the height and width of the images, along with the proxy URLs